spec/stat_tracker_spec.rb

```diff
@@ -168,4 +168,110 @@
       expect(@stat_tracker2.count_of_games_by_season['20142015']).to eq(0)
     end
   end
+
+  describe '#team_shot_goal_ratio' do
+    it 'calculates accuracy ratio of a team' do
+      @stat_tracker1.team_shot_goal_ratios
+      expect = {
+        "Atlanta United"=>0,
+        "Chicago Fire"=>0,
+        "FC Cincinnati"=>0,
+        "DC United"=>0,
+         "FC Dallas"=>0.32,
+        "Houston Dynamo"=>0.21,
+         "Sporting Kansas City"=>0.06,
+      }
+      expect(@stat_tracker1.team_shot_goal_ratios).to include(expect)
+    end
+
+    it 'calculates accuracy ratios for a specific season' do
+      @stat_tracker1.team_shot_goal_ratios('20122013')
+      expect = {
+        "Atlanta United"=>0,
+        "Chicago Fire"=>0,
+        "FC Cincinnati"=>0,
+        "DC United"=>0,
+        "FC Dallas"=>0.32,
+        "Houston Dynamo"=>0.21,
+      }
+      expect(@stat_tracker1.team_shot_goal_ratios('20122013')).to include(expect)
+    end
+  end
+
+  describe '#most_accurate_team' do
+    it 'shows the team with the highest goal average' do
+      @stat_tracker1.most_accurate_team
+      expect(@stat_tracker1.most_accurate_team).to eq("FC Dallas")
+    end
+
+    it 'shows most accurate team in the season' do
+      @stat_tracker1.most_accurate_team('20122013')
+      expect(@stat_tracker1.most_accurate_team('20122013')).to eq("FC Dallas")
+    end
+  end
+
+  describe '#least_accurate_team' do
+    it 'shows the team with the lowest goal average' do
+      @stat_tracker1.least_accurate_team
+      expect(@stat_tracker1.least_accurate_team).to eq("Atlanta United")
+    end
+  end
+
+  describe '#least_accurate_team by season' do
+    it 'shows least accurate team in the season' do
+      @stat_tracker1.least_accurate_team('20122013')
```

Comment:
you shouldn't need this line, the next line should be handling everything it needs